cloudformation/kms.yaml

```diff
@@ -21,22 +21,13 @@ Resources:
           - Sid: 'Enable IAM Root Permissions'
             Effect: Allow
             Principal:
-              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
+              AWS: !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:root'
             Action: 'kms:*'
             Resource: '*'
-          - Fn::If:
-              - isDev

```

Comment:
@nguyen102 If this is removed will we be able to use KMS when we do `sls offline`?

Reply:
The user role that is running `sls offline` should still be able to access the S3 bucket if the user role has KMS access permission. So if you're using an admin role, then `sls offline` should still work.

It looks like this KMS key already provide access to the root account.
More info here.
https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html